backend/src/main/java/org/example/log/LogExecution.java

```diff
@@ -0,0 +1,11 @@
+package org.example.log;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public  @interface LogExecution {
```

Comment:
로그 실행에 관련된 부분 잘 해주셨습니다!!
이 부분은 "로그 파일 기록"에 대한 어노테이션이므로, `@LogToFile`로 바꾸는 것을 제안 드립니다

```suggestion
public  @interface LogToFile {
```